submissions/SanaAbuNada/assignment8.js

```diff
@@ -0,0 +1,66 @@
+
+// Task1
+
+const editor = {
+    title: 'my first blog',
+    getUpperTitle() {
+    return this.title.toUpperCase();
+}
+};
+// Write code that ensures getTitle() works as expected and logs "MY FIRST BLOG"
+
+const getTitle = editor.getUpperTitle.bind(editor);
+console.log(getTitle());
+
+// Task2
+
+const formHandler = {
+    value: 'initial',
+    onChange :(newValue)  => {
```

Comment:
u should not edit the original object

Reply:
also, this is wrong as we said, using `this` inside arrow function as an object method directly will result in undefiend